test-runner-frontend/src/Views/ReportUtils.js

```diff
@@ -0,0 +1,86 @@
+import html2canvas from 'html2canvas';
```

Comment:
can we please add some tests for this file? its a little difficult to understand what data shapes some of these functions expect without tests

Reply:
Done.